prod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8.16.0-alpine as builder
+
```

Comment:
For compatibility with `niployments`, the file should be renamed to `Dockerfile-prod`.